### DISCUSSION TITLE
Enhancement: option to ask question when registering on the waiting list.

### DISCUSSION THREAD

**Discussion by HansF** [pretix/pretix#2688] [Ideas] [answered]
Dear, 

**Thanks** again for this awesome software. If I can share my 2 cents, I'd like the option to add a question to the signup form for the waiting list. 

In this specific case I want to ask people for their 'forum nickname'. Another question I'd like to ask is a bit about their motivation for joining the event.

I'd prefer to see the the answers to the questions in the table of the waiting list, but if they are included when exporting to csv that would be fine. Alternatively, if there is some way to click on a waiting list entry and see the answers that would be awesome as well.

The questions can be from the same menu of the products > questions. I propose to add a checkbox "_ask this question when joining the waiting list_" to the "advanced" tab of the edit question page.

**Comment by raphaelm** [comment, marked as answer]
Hmm, I'm on the edge about this idea, it sounds quite nice, but I'm not sure it warrants the complexity and I'm not sure an additional checkbox in the questions dialog is the right way to go, we'd probably need to redesign the question dialog in some way (an additional checkbox would eg make it unclear what happens if multiple of this "ask this… when…" boxes are ticked/unticked).

**Reply by HansF** [reply]
Thanks for your comment, I understand the points you are making. 

Would there be an option to ask the same questions when signing up for the waitinglist for a specific ticket as when purchasing the ticket? Meh, come to think of it, I agree on the complexity you mentioned, and maybe it's best to KISS. 

I went to take a look at the waitinglist screen: 
![image](https://user-images.githubusercontent.com/1503/174100372-6d61e321-84c3-450d-a3af-20621b75098d.png)

Maybe it's an option to drop the predefined name and phone number question there, and have the ability to add a number of 'single line text'-questions there?

**Reply by raphaelm** [reply]
No, dropping these would actually be worse, as we'd loose eg phone number validation and the option for SMS sending on plugins, tex

**Reply by HansF** [reply]
Thanks for the clarification!
